dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java

```diff
@@ -67,7 +67,11 @@ public void testSearchOperations() throws ApiException {
         ExtendedGa4GhApi extendedGa4GhApi = new ExtendedGa4GhApi(webClient);
         // update the search index
         extendedGa4GhApi.toolsIndexGet();
-
+        try {
```

Comment:
Suggest a comment on why you are doing these Thread.sleep()'s. Or maybe since you do it 3 times, a well-named method that does the sleeps, e.g., something like `waitForElasticSearchChangesToFlush|Propagate|Refresh`.